### DISCUSSION TITLE
Considering offering a set of TypeScript interfaces for Phantom's APIs

### DISCUSSION THREAD

**Discussion by nbran** [phantom/docs#219] [General]
This would go a long way in improving the quality of Phantom integrations and generally provide a better understanding of the APIs.

**Comment by adamdelphantom** [comment]
@nbran This request would be a good candidate for our [feature request board.](https://phantom.canny.io)

The more detail you can provide of how this would help you as a developer, the better. And if you can get other developers to upvote the request that will help drive the case for it to be offered.
